src/sgame/sg_svcmds.cpp

```diff
@@ -269,6 +269,68 @@ static gclient_t *ClientForString( char *s )
 	return &level.clients[ idnum ];
 }
 
+/*
+===================
+Svcmd_EntityLock_f
+===================
+*/
+
+static void Svcmd_EntityLock_f()
+{
+	char      argument[ 4 ];
+	int       e;
+	gentity_t *door;
+	vec3_t    mins, maxs;
+
+	if ( trap_Argc() < 1 )
+	{
+		Log::Notice( "usage: entitylock <entityNum> (tip: use " 
+		             "'cg_drawEntityInfo on' on your client to find this)" );
+		return;
+	}
+
+	trap_Argv( 1, argument, sizeof( argument ) );
+
+	if ( !atoi( argument ) )
```

Comment:
you could store the value into `e` first and then check that `e == 0` it would avoid useless computations.